tests/conftest.py

```diff
@@ -89,6 +96,16 @@ def script_file():
     In order for the factory to be able to extract the function body properly,
     function header ("def") must all be on a single line, with nothing after
     the colon but whitespace.
+
+    Specific lines can be marked with a comment like so:
+
+        print('fizz') #@fizz
+
+    The returned string value has an additional attribute 'lines', which is
+    a dict of all marked lines - the part after #@ becomes the key, and the
+    corresponding line number is the value. This can be used to set breakpoints
+    or to assert line numbers for exceptions and stack traces in a way that
+    does not require updating them as debugged code changes.
```

Comment:
@karthiknadig, check this out, I think this'll come in handy in quite a few tests.

Reply:
Just realized that this approach won't work for tests that use named files. Better to have it as a separate function that opens a file by name and extracts that info, then it's fully reusable.